stylesheets/components/input.css

```diff
@@ -7,18 +7,17 @@
     white-space: nowrap;
     border-radius: var(--border-radius);
     border: 1px solid var(--border-color);
-    box-shadow: 0 1px 2px rgba(2, 36, 102, .15) inset;
     background-color: var(--input-background);
     color: var(--input-color);
 }
 
 .input--small {
-    font-size: var(--font-size--small);
+    font-size: .95em;
```

Comment:
Interesting... 🤔 Any (new) guidelines around using ems?

Reply:
this one comes from visual perception: default small size of 0.8em looks too small in the input box. 

The visual difference between regular and small input comes mostly from decreased paddings. 
I couldn't find any actual UBIO designs that use smaller inputs so you can look at this one as a draft. Feel free to change/request a change as soon as it becomes clear that small input should look different. Our base font size (12px) is quite small already, and it's not a common practice to use smaller font sizes in inputs on the web (it's usually something ok-readable as 12px on desktop or 16px on mobile)